.schema/version.schema.json

```
@@ -0,0 +1,22 @@
+{
+  "$id": "https://github.com/ory/kratos/.schema/versions.config.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

One small thing is missing: this should be
```suggestion
  "$schema": "https://raw.githubusercontent.com/ory/cli/{{ release }}/.schema/version_meta.schema.json#",
```
but we have to release the cli therefore.